common/v2/services/EthService/nodes/web3/client.ts

```diff
@@ -10,6 +10,7 @@ export default class Web3Client extends RPCClient {
     this.provider = (window as any).web3.currentProvider;
   }
 
+  // @ts-ignore
```

Comment:
This should be written so it doesn't require `@ts-ignore`.

Reply:
The error appeared after moving the files and without changing the types. I wasn't able to find a solution for it, and seems to require a rework of type declarations. Which is out of the scope of this PR. I would prefer to do it later.